### PR TITLE
heartcircle

### PR DESCRIPTION
**Pull request by benpepin**

### CODE REVIEW

File: source/index.html.erb [hunk 1]
```diff
@@ -49,6 +49,17 @@ title: browserstats.love
...
 <section class="credits">
+
+  <div class="heart-circle">
+    <div class="heart">
+      ❤️
+
+
+    </div>
+
+
```
Comment (tobysilverman) [inline source/index.html.erb]: :art:
Why so much extra whitespace?

File: source/stylesheets/site.css [hunk 2]
```diff
@@ -119,6 +119,21 @@ td.browser-column {
   the credits
 */
 
+.heart-circle {
+  width: 72px;
+  height: 72px;
+  border-radius: 50px;
```
Comment (tobysilverman) [inline source/stylesheets/site.css]: 💭 
did you mean 50%?

File: source/stylesheets/site.css [hunk 3]
```diff
@@ -119,6 +119,21 @@ td.browser-column {
...
+  background-color: white;
+  margin: auto;
+  box-shadow: 0 1px 2px 0px rgba(0, 0, 0, 0.15);
```
Comment (tobysilverman) [inline source/stylesheets/site.css]: please keep with designer specs -- should be `0 1px 2px 0px rgba(0, 0, 0, 0.25);`

File: source/index.html.erb (hunk 1 above)
Comment (tobysilverman) [reply source/index.html.erb]: `<div class="heart">❤️</div>`

**Review by tobysilverman — CHANGES_REQUESTED** [review]

**Comment by tobysilverman** [comment]
Nice dawg--

Love it.

File: source/index.html.erb (hunk 1 above)
Comment (benpepin) [reply source/index.html.erb]: Done

File: source/stylesheets/site.css (hunk 2 above)
Comment (benpepin) [reply source/stylesheets/site.css]: Done

File: source/stylesheets/site.css (hunk 3 above)
Comment (benpepin) [reply source/stylesheets/site.css]: matched it with the top bar.  guess that's wrong too - you should tell whoever did that to change it.

done

**Comment by strickinato** [comment]
Changes look good to me @tobysilverman 

Go ahead and merge

**Review by tobysilverman — APPROVED** [review]

**Pull request merged**